### DISCUSSION TITLE
Displaying WooCommerce Notices on Elementor Popups

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#25712] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

As you know, we can now customize WooCommerce notice. But it would be great if we could display them as a popup. This is an important issue. Please review it.

### Describe the solution you'd like

Displaying WooCommerce notice as popup (elementor popup)

**Comment by nicholaszein** [comment]
Hey @Michaelgimii,

thanks for submitting your feature request! 🙏 

We will keep this suggestion in mind and consider adding it if there is significant demand in the future.  

Best regards

**Comment by webfromdev** [comment]
Hey guys, I would also like to add this popup notification to my woocommerce ❤️

**Comment by nicholaszein** [comment]
Hi there,

### ✈️ This Feature Request has been migrated from issue #20424 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Happy voting! 🙂

**Comment by nicholaszein** [comment]
This Request was mentioned in the following issues and discussions:

- #18628
- #21623
